packages/webapp/src/utils/credits-purchase-constants.js

```diff
@@ -1,12 +1,20 @@
 const routes = {
-  ESTIMATOR_CREDITS_INDIVIDUAL_ORG: '/credits-purchase/applying-individual-organisation'
+  // CREDITS_PURCHASE_APPLYING_INDIVIDIAL_ORGANISATION: '/credits-purchase/applying-individual-organisation',
+  CREDITS_PURCHASE_CHECK_DEFRA_ACCOUNT_DETAILS: '/credits-purchase/check-defra-account-details'
+  // REGISTER_CREDIT_PURCHASE_TASK_LIST: '/credit-purchase/register-credit-purchase-task-list'
```

Comment:
Happy to leave this line commented until https://github.com/DEFRA/biodiversity-net-gain-service/pull/615 is merged.

Reply:
done